src/preload.py

```diff
@@ -505,40 +673,56 @@ def get_docker_storage_driver(image=None):
             position = find_one_of(words, "-s", "--storage-driver")
             if position != -1 and position < len(words) - 1:
                 return words[position + 1]
-    # Stop here if no driver was found
-    assert False
+    assert False, "Docker storage driver could not be found"
 
 
-def main_preload(app_data, container_size, detect_flasher_type_images):
+def main_preload(app_data, container_size):
     # Size will be increased by 110% of the container size
-    additional_space = round_to_sector_size(ceil(int(container_size) * 1.1))
-    inner_image = get_inner_image_filename()
-    if not detect_flasher_type_images and inner_image:
-        log.info(
-            "Warning: This looks like a flasher type image but we're going to "
-            "preload it like a regular image."
-        )
-    if detect_flasher_type_images and inner_image:
-        part = get_partition("root")
-        log.info(
-            "This is a flasher image, preloading into /opt/{} on the 2nd "
-            "partition of {}".format(inner_image, part["image"])
-        )
-        resize_rootfs(additional_space)
-        with mount_context_manager("root") as mountpoint:
-            image = os.path.join(mountpoint, "opt", inner_image)
-            preload(additional_space, app_data, image)
+    additional_bytes = round_to_sector_size(ceil(int(container_size) * 1.1))
```

Comment:
What is the `1.1` magic number?

Reply:
It's the 110% from the comment on the line above:
>Size will be increased by 110% of the container size

Reply:
Ah yeah, the diff made that quite hard to see :)